main.go

```diff
@@ -27,6 +27,7 @@ var Options struct {
 	RHCOSVersions         string `envconfig:"RHCOS_VERSIONS"`
 	OSImages              string `envconfig:"OS_IMAGES"`
 	AllowedDomains        string `envconfig:"ALLOWED_DOMAINS"`
+	InsecureSkipVerify    string `envconfig:"INSECURE_SKIP_VERIFY"`
```

Comment:
This can be a bool with default set to `false`.
Here's an example from assisted-service: https://github.com/openshift/assisted-service/blob/80170758e2463f3b0f7e1c0256f1ebf7be57d0ab/cmd/main.go#L146

Reply:
Modified, thank you for the example!